cmd/mimir/config-descriptor.json

```diff
@@ -2116,20 +2116,42 @@
               "kind": "field",
               "name": "failure_threshold",
               "required": false,
-              "desc": "Max number of requests that can fail in a row before the circuit breaker opens",
+              "desc": "Max percentage of requests that can fail over period before the circuit breaker opens",
               "fieldValue": null,
               "fieldDefaultValue": 10,
               "fieldFlag": "ingester.client.circuit-breaker.failure-threshold",
               "fieldType": "int",
               "fieldCategory": "experimental"
             },
+            {
+              "kind": "field",
+              "name": "failure_execution_threshold",
+              "required": false,
+              "desc": "How many requests must have been executed in period for the circuit breaker to be eligible to open for the rate of failures",
+              "fieldValue": null,
+              "fieldDefaultValue": 100,
+              "fieldFlag": "ingester.client.circuit-breaker.failure-execution-threshold",
+              "fieldType": "int",
+              "fieldCategory": "experimental"
+            },
+            {
+              "kind": "field",
+              "name": "period",
```

Comment:
Maybe we could call this thresholding_period, or failure_thresholding_period?

Reply:
Sure, either of those works for me. Do you have a preference?

Reply:
Went with `thresholding_period`